app/src/main/java/org/systers/mentorship/view/fragments/RequestPagerFragment.kt

```diff
@@ -45,7 +45,34 @@ class RequestPagerFragment: BaseFragment() {
             requestsList = it.getParcelableArrayList(Constants.REQUEST_LIST)
             emptyListText = it.getString(Constants.REQUEST_EMPTY_LIST_TEXT)
         }
+        setView()
+    }
+
+    private val openRequestDetail: (Relationship) -> Unit =
+            { requestDetail ->
+                val intent = Intent(activity, RequestDetailActivity::class.java)
+                intent.putExtra(Constants.RELATIONSHIP_EXTRA, requestDetail)
+                startActivityForResult(intent, Constants.DELETE_REQUEST_RESULT_ID)
+            }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        val id = data?.getStringExtra(Constants.REQUEST_ID)
+        val iter = requestsList.iterator()
+
+        while (iter.hasNext()) {
+            val reqId = iter.next().id
+            if(reqId.toString() == id) {
+                iter.remove()
+            }
+        }
+        setView()
+    }
 
+    /**
```

Comment:
Can you please remove these comments? @shiv07tiwari Codacy is failing for this minor issue. After this, I can merge this :)